packages/core/src/devTools/redux.ts

```diff
@@ -0,0 +1,39 @@
+import { State } from '../State';
```

Comment:
its pretty small but also its not part of the core, so maybe worth moving into a separate package? if you think that would be preferable I can take care of creating a new directory and wiring things up

Reply:
For now, until we have solid dev tools, let's keep this here. It won't be included in core, but should be able to be imported from here still.